tests/runtest.sh

```diff
@@ -370,6 +370,8 @@ function create_core_overlay {
         rm -f "$coreOverlayDir/System.Private.CoreLib.ni.dll"
     fi
     copy_test_native_bin_to_test_root
+
+    chmod +x $coreOverlayDir/corerun
```

Comment:
I don't think this is necessary here. We do it in corefx because we get the corerun / dotnet from a nuget package and that somehow doesn't preserve the executable bit. But here, the corerun should be comming from the currently built stuff and so it should be marked as executable, I believe.